scripts/us_census/pep/us_pep_sexrace/README.md

```diff
@@ -0,0 +1,126 @@
+# US Census PEP: Populations Estimates By Sex And Race
```

Comment:
There are a few .DS_Store files checked in. Please remove those files, and add .DS_Store to .gitignore so that these files won't be included in PR in the future.

Reply:
Removed .DS_Store.